### ISSUE TITLE
Opening a saved .FPM Level while the Object Library is open will cause the level to load with nothing in it. 

### ISSUE THREAD

**Issue by Kitakazi** [TheGameCreators/GameGuruRepo#4815]
Click the Add button to open the Object Library. 
Next, while the Object Library is open, go to File > Open Level, and select a previously saved FPM. 
The FPM will load with no entities on the level.

**Comment by LeeBamberTGC** [comment]
@Kitakazi In the latest EXP build I could bounce between two levels with the Add Object library window open:
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/9e415f5b-f121-4c65-b096-5b655da589cb)


Can you make a short video showing what you see, it might be the folder you are viewing or the window UI setting.

**Comment by Kitakazi** [comment]
Sure here is a video. https://youtu.be/74pppqFyx0U
Same results for any level, only particles and other game elements are loaded all other entities are missing.

**Comment by LeeBamberTGC** [comment]
@Kitakazi Thanks for the video, I was able to reproduce with the Aztec Teaser from a fresh run and opening the library. I also spotted that pretty much ALL the menu items are not appropriate when the object library window is open, so taken the extra step of basically denying access to the MENU when the object library window is being used.  In the next EXP build, there should be no way of opening a level while you are viewing the object or any other library.
